runelite-client/src/main/resources/net/runelite/client/plugins/chatfilter/default_chat_filter.txt

```diff
@@ -0,0 +1,68 @@
+(?i)^doublin.*?
```

Comment:
I don't think this should be included in this pr

Reply:
this line in particular or the whole file?

I wanted to add a community-managed base filter that would cover most cases of G.E. spamming so that new users wouldn't have to manually add all the initial regex's themselves

Reply:
I agree this shouldn't be done.

1) it can block messages the user doesn't consider to be spam
2) it gives the bad guys a known good list of filters to avoid, which would render it useless anyway soon

Someone on Discord was working on a NN that could identify spam which I think is the correct approach for this problem if we wanted a more automated way of classifying spam.

Reply:
alright, I'll get rid of the default filter option

Reply:
@Adam- the NN solution sounds interesting, is there a link to an issue tracking it?

Reply:
I'm working on the NN solution, using deep learning to train a spam detection model on some training data manually collected from the GE. The model itself works well enough for the moment, currently working on implementing the model properly in RL. My code is mostly local for now since I'm still messing with model structure. Let me know if you're interested in learning more or helping!

Reply:
@naallen that sounds super cool, look forward to seeing it in the future!